db/migrate/20150709205405_create_k12_theme.rb

```diff
@@ -33,7 +34,11 @@ def up
     bc = BrandConfig.new(variables: variables)
     bc.name = NAME
     bc.share = true
+    bc.md5 = MD5_OF_K12_CONFIG
+    # set the md5 directly in the record for backwards compatability
+    BrandConfig.skip_callback :validation, :before, :generate_md5
```

Comment:
you should use suspend_callbacks instead

Reply:
That's a good way to do it.  I pushed a change, but it's not right.  I'll get this fixed

Reply:
Should be fixed now.